Makefile

```diff
@@ -20,6 +20,9 @@ endif
 
 build_dir = 'tmp/_output'
 
+compile:
```

Comment:
Add "compile" to the .PHONY list at the bottom.

Reply:
- [x] Done